.github/workflows/ci-a11y.yml

```diff
@@ -0,0 +1,63 @@
+name: Accessibility tests
+
+on:
+  pull_request:
+    paths:
+      - app/**
+      - .github/workflows/ci-a11y.yml
+
+jobs:
+  build:
+    name: Pa11y-ci tests
```

Comment:
name the job based on what it's doing not the tool it's using e.g. "Accessibility" not "Pa11y"